### DISCUSSION TITLE
Where to paste a ETA template?

### DISCUSSION THREAD

**Discussion by xuey06** [retorquere/zotero-better-bibtex#2456] [Q&A] [answered]
I found a ETA template on zotero forum to realize the Quick Copy of an item with the form of [@citekey](zotero link).
When I select the format of Quick-Copy as the 'Eta template' in the preference of BBT, I can not find the place to paste the ETA template I got.
So,  where to paste a ETA template?
Thank you for helps.

**Comment by retorquere** [comment, marked as answer]
That's a bug, can you open an issue for that?

**Reply by xuey06** [reply]
Sure, thanks for reply.
